src/base/bittorrent/session.h

```diff
@@ -52,6 +52,11 @@
 #include "torrentinfo.h"
 #include "trackerentry.h"
 
+namespace libtorrent
+{
+    struct torrent_conflict_alert;
```

Comment:
> libtorrent didn't forward declare it?

Exactly!
https://github.com/arvidn/libtorrent/issues/7016

Reply:
Since libtorrent has patched it up, it would be nice to add a TODO comment about removing it when libtorrent >= 2.0.8.
Also it should be enclosed in `#ifdef QBT_USES_LIBTORRENT2`

Reply:
> Since libtorrent has patched it up, it would be nice to add a TODO comment about removing it when libtorrent >= 2.0.8. Also it should be enclosed in `#ifdef QBT_USES_LIBTORRENT2`

Done.